lib/src/network/network.spec.ts

```diff
@@ -23,16 +57,62 @@ describe('Network', () => {
             expect(() => {
                 // Non-hex characters
                 Devnet({
+                    feeConfig: ANY_FEE_CONFIG,
                     chainHexId: 'ZZ' as any,
                 });
             }).to.throw(
                 'Expected property value to be two hex characters of chain Id\n- Expected `chainHexId` to be of type `Buffer` but received type `string` in object `options`',
             );
         });
 
+        it('should throw Error when Linear fee constant and/or coefficient is invalid', () => {
+            expect(() => {
+                // eslint-disable-next-line no-new
+                Devnet({
+                    feeConfig: {
+                        algorithm: FeeAlgorithm.LinearFee,
+                        constant: new BigNumber('-1'),
+                        coefficient: new BigNumber('1001'),
+                    },
+                    chainHexId: 'AB',
+                });
+            }).to.throw(
+                'Expected property property value to be greater than or equal to 0 in object `feeConfig` in object',
```

Comment:
"property" mentioned twice though nothing major in this.

Reply:
It appears that the `ow` argument validation library we are using is not smart enough in handling nested validation error message. Hence resulting something like this.

I will open a separate issue to try to reproduce and see if I can fix in their library easily.